lib/smart-survey-api.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const axios = require('axios');
+const querystring = require('querystring');
+const SmartSurveyClient = require('smartsurvey-client');
+
+module.exports = {
+  getResponses: (baseUrl, surveyID, endPoint, token, tokenSecret) => {
+    querystring.stringify({ foo: 'bar', baz: ['qux', 'quux'], corge: '' });
+    const apiUrlPart = querystring.stringify({'api_token': token, 'api_token_secret': tokenSecret});
+    axios.get(`${baseUrl}/${surveyID}/${endPoint}?${apiUrlPart}`)
```

Comment:
thanks @chrisns 
I'm not sure how to augment/extend it, is there any harm accepting this PR as it is and then I can do a separate PR for what you are suggesting? 
I'm just wondering if this PR is just getting bigger and bigger than what it needs to be?